multigtfs/models/route_direction.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2012-2014 John Whitlock
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import unicode_literals
+
+from django.utils.encoding import python_2_unicode_compatible
+from jsonfield import JSONField
+
+from multigtfs.models.base import models, Base
+
+
+@python_2_unicode_compatible
+class RouteDirection(Base):
+    """Associate a name with a Route and Direction
+
+    Maps to non-standard route_directions.txt in GTFS feed.
```

Comment:
Are there any online docs for the format of this file that we could link here?

Reply:
Maybe the best approach would be for me to create a section for it under `Optional Files` in `docs/gtfs.rst`? I could do that this afternoon and add to this PR.

The closest thing to documentation seems to be [this discussion](https://groups.google.com/forum/#!topic/gtfs-changes/Dx9nRwQo6rQ), which mainly discusses adding this info to trips.txt, but the final paragraph of the final post in that thread explains why a `route_directions.txt` would be a better approach and seems to be where it was first suggested. Guessing people ran off with it after that.